configure.ac

```diff
@@ -56,6 +56,10 @@ AC_ARG_WITH([mpi],
              LDFLAGS="-L$withval/$mpi_libdir $LDFLAGS"],
             [])
 
+AC_ARG_ENABLE([tests], [AS_HELP_STRING([--disable-trace],
+      [Disable build of test binaries])], [enable_tests=$enableval], [])
```

Comment:
nit, but you don't need the `enable_tests=$enableval` argument.  `enable_tests` will be set to `$enableval` by default if the 3rd argument to AC_ARG_ENABLE is not provided.  So this can become:

```
AC_ARG_ENABLE([tests], [AS_HELP_STRING([--disable-trace],
      [Disable build of test binaries])])
```